src/backend-java-quarkus/src/main/java/ai/azure/openai/rag/workshop/backend/configuration/ChatLanguageModelAzureOpenAiProducer.java

```diff
@@ -13,9 +14,6 @@ public class ChatLanguageModelAzureOpenAiProducer {
 
   private static final Logger log = LoggerFactory.getLogger(ChatLanguageModelAzureOpenAiProducer.class);
 
-  @ConfigProperty(name = "AZURE_OPENAI_KEY", defaultValue = "__dummy")
```

Comment:
Please revert this and the change below as it will break support for the OAI proxy and running the project locally, as managed identity doesn't work in containers.

Instead you can use the same approach as in the Node.js version: https://moaw.dev/workshop/gh:sinedied/openai-rag/docs/workshop-qdrant.md?step=7#managing-azure-credentials

This will keep compatibility with the proxy, and use managed identity when deployed or running outside the container.